src/cli/Cli.spec.ts

```diff
@@ -47,6 +47,33 @@ class MockCommand extends AnonymousCommand {
   }
 }
 
+class MockCommandWithOptionSets extends AnonymousCommand {
+  public get name(): string {
+    return 'cli mock optionsets';
+  }
+  public get description(): string {
+    return 'Mock command with option sets';
+  }
+  public options(): CommandOption[] {
+    const options: CommandOption[] = [
+      {
+        option: '--opt1 [name]'
+      },
+      {
+        option: '--opt2 [name]'
+      }
+    ];
+    const parentOptions: CommandOption[] = super.options();
+    return options.concat(parentOptions);
+  }
+  public optionSets(): string[][] | undefined {
+    return [["opt1", "opt2"], []];
```

Comment:
Why are we defining here an empty option set? It's not something we'd ever do in a command